lib/rspec/hanami/match_type.rb

```diff
@@ -0,0 +1,18 @@
+module RSpec
+  module Hanami
+    class MatchType
+      def call(actual, expected)
+        actual.class == expected
+      end
+    end
+  end
+end
+
+module Type
```

Comment:
WDYT if we use `Hanami::Entity::Types` instead this module?
![screenshot 2018-07-27 12 36 34](https://user-images.githubusercontent.com/1147484/43313789-c2f16436-9199-11e8-8908-35536fe04e10.jpeg)